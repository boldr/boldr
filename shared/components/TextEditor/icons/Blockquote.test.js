import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Blockquote from './Blockquote';

describe('<Blockquote />', () => {
  const wrapper = shallow(<Blockquote />);
  it('renders <Blockquote /> without breaking', () => {
    expect(wrapper.find('svg').length).toBe(1);
  });
  it('renders snapshot', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
