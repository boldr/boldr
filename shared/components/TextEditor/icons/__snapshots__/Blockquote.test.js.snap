exports[`<Blockquote /> renders snapshot 1`] = `
<svg
  height="18"
  viewBox="0 0 18 18"
  width="18">
  <path
    d="M10 5v5h2.75L11 13h2.25L15 10V5h-5zm-7 5h2.75L4 13h2.25L8 10V5H3v5z" />
</svg>
`;
